cmd/tsgo/main.go

```diff
@@ -295,7 +298,44 @@ func runMain() int {
 	}
 	stats.add("Total time", totalTime)
 
-	stats.print()
+	if opts.devel.extendedDiagnostics != "" {
```

Comment:
This isn't really the same thing as the original tsc flag; I would not personally want to add this flag, at least not under this name.

Why not just parse out the current output, which is what needs to be done with old tsc as well?

Reply:
> Why not just parse out the current output, which is what needs to be done with old tsc as well?

For exactly that reason, it's pretty annoying bit of code to maintain. Happy to think of other names! The context for me is that we send this information to Datadog, so it's pretty nice to not have to maintain that code.

Reply:
Needing to send this information somewhere is making me more nervous than before. If we add `devel` flags here, they're temporary until we can discuss and agree on a real flag that will also be present in tsc. tsgo itself is going to go away and be replaced tsc (with extra flags), so I would be wary about adding a flag that we'll just remove later...